src/main/resources/liquibase/crop_changelog/16_4_0.xml

```diff
@@ -221,4 +221,17 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="clarysabel" id="v16.4.0-9">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<indexExists tableName="germplsm" indexName="germplsm_idx14"></indexExists>
+			</not>
+		</preConditions>
+		<comment>Add index for germplsm gid and germplsm_uuid columns</comment>
+		<createIndex indexName="germplsm_idx14" tableName="germplsm" unique="false">
+			<column name="gid"></column>
+			<column name="germplsm_uuid"></column>
+		</createIndex>
+	</changeSet>
+
```

Comment:
Looks fine. Did you have time to investigate in which way it improves the performance? The only clause I can think of is the [`OR`](https://github.com/IntegratedBreedingPlatform/Middleware/pull/935/files#diff-70aa9494d2c25b3565e0b4a4580ccb910bd79b83399e6f82bf718d5c6ea1176aR1786) case, but it's strange because I would expect it to use the normal indices in that case. But if it works..